neovim/api/buffer.py

```diff
@@ -98,17 +98,17 @@ def range(self, start, end):
         return Range(self, start, end)
 
     def add_highlight(self, hl_group, line, col_start=0,
-                      col_end=-1, src_id=-1, async=None):
+                      col_end=-1, src_id=-1, _async=None):
         """Add a highlight to the buffer."""
-        if async is None:
-            async = (src_id != 0)
+        if _async is None:
+            _async = (src_id != 0)
         return self.request('nvim_buf_add_highlight', src_id, hl_group,
-                            line, col_start, col_end, async=async)
+                            line, col_start, col_end, _async=_async)
```

Comment:
Does the parameter really need to be changed? That's silly, python parser should know the difference. 

This will be a breaking change for our users

Reply:
I machinery replaced all `async` into `_async` so I'm not sure about this particular line.
However the screenshot I pasted showed `invalid syntax` on function attributes and this PR fixed that issue.